### DISCUSSION TITLE
Create new application credential but api return unauthorized

### DISCUSSION THREAD

**Discussion by simpsons310** [emqx/emqx#9768] [Q&A] [answered]
I'm trying to use EMQX api to get client data, but api basic authentication only work for default credentials (`admin/public`) (if create new appId and appSecret, these not work and return unauthorized). Following is steps I've done:

- Enviroment information
  + EMQX version: 4.4.4 via docker image (https://hub.docker.com/layers/emqx/emqx/4.4.4/images/sha256-3b692009920168cd2bbb21b55d22d94d38f4fbb5569abd38099f670480a6cd72?context=explore)

- Steps reproduce:
  + Call api get client info via username: `/api/v4/clients/username/{client_username}` with default credentials `admin/public` => worked as deserved
  + Create new application credential in `General > Applications`
![image](https://user-images.githubusercontent.com/34018204/212620912-f12c804c-9bb5-422b-b8df-3f1f5c379c4a.png)
  + Call api get client info via username with new AppID and AppSecret => return 401 code with content `UNAUTHORIZED`

Can someone tells me what I was wrong or this is a bug on this version

**Comment by lafirest** [comment, marked as answer]
@simpsons310 
I guess maybe you have called the API with port 18083, but in v4 this port is just for the dashboard.
The default port for HTTP API is 8081, you can retry your steps by only changing the port to 8081.

**Reply by simpsons310** [reply]
@lafirest Tried port 8081 and worked perfectly. Thanks
